app/scripts/services/unit.coffee

```diff
@@ -347,6 +347,17 @@ angular.module('swarmApp').factory 'Unit', (util, $log, Effect, ProducerPaths, U
     #util.assert max.greaterThanOrEqualTo(0), "invalid unit cost max", @name
     return max
 
+  _costMetPerSecond: ->
+    max = new Decimal Infinity
+    console.log max
+    for cost in @eachCost()
+      if cost.val.greaterThan(0)
+        console.log cost,cost.val.toNumber(),cost.unit.velocity().toNumber(),cost.unit.velocity().dividedBy cost.val
```

Comment:
use `$log.debug` instead, so this doesn't appear in production. same with all other `console.log`s.

it'd be helpful to have some sort of fixed message to tell where the message is coming from too, like `$log.debug '_costMetPerSecond', ...`